flamegraph.rb

```diff
@@ -0,0 +1,31 @@
+#
+# Read perf.data using FlameGraph
+# Run this script only when `perf record` is properly terminated (a.k.a: ruby profile_node.rb)
+# - Copy perf.data generated in Deku to FlameGraph folder* 
+# - Go inside FlameGraph folder and use `perf script` to read the information inside
+# perf.data
+# - Use flamegraph commands and save the result in picture as `perf-flamegraph.svg`
+# - Wait for awhile to let the graph generated, you can use internet browser to see the file
+# `perf-flamegraph.svg`:
+#  + firefox perf-flamegraph.svg # or chrome, etc.
+#
+# Note:
+# FlameGraph need to be downloaded at:
```

Comment:
I am not sure if it is a good idea to include the code of flame-graph inside, because it can be problematic to maintain. 
cc: @aguillon what do you think about it?

Reply:
I think including the code directly is a bad idea (for licensing and versioning reasons). It seems that it is a single file with no dependency, right? You can download it automatically pretty easily if it's missing from your main script.